### ISSUE TITLE
Orientation of names in chromosome wheel

### ISSUE THREAD

**Issue by AlistairNWard** [chmille4/bam.iobio.io#50]
Do we want these to be rotated?

![screen shot 2016-09-20 at 9 09 45 am](https://cloud.githubusercontent.com/assets/523111/18671466/34f2daa2-7f12-11e6-9676-0b1e08ec34e6.png)

**Comment by chmille4** [comment]
Yes, you can fit longer names this way. For example sometimes the ref names
are chr1 instead of 1. Chr1 will not fit unless it's rotated

**Comment by AlistairNWard** [comment]
Is it possible to have them oriented correctly though? chr 8+ look good, but would be better if 1-7 were not upside down!
